### ISSUE TITLE
Home page card links look like headings

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#229]
The card links on the homepage under 'Key elements' look like headings but are no headings.
This fails WCAG 1.3.1 Info and Relationships.

This can be a bit subjective. But when we discussed this among specialists we all agreed they look too much like headings.

Another concern is that there is nothing semantic grouping the link and the text underneath together. That could be solved by putting them into a list, but making the link a heading would also solve that.
